File: lists/perfect_square.js


const perfect = () => {
    let result = [];
    for (let x = 1; x <= 20; x++) {
        let temp = `${x} * ${x} = ${x * x}`;
        result.push(temp);
    }

    for (let i in result) {
        console.log(result[i]);
    }
}


perfect();

File: lists/perfect_square.py
import math 

def perfect():
    result = []
    x = 1

    while len(result) < 20:
        root = math.sqrt(x)
        if root % 1 == 0 and root * root == x:
            result.append("{} * {} = {}".format(int(root), int(root), x))

        x += 1
    
    for r in result:
        print(r)

perfect()